src/components/resolvers/Rinkeby/0x2930Cf9EE8E03C3E06Fa1828cCD8E371323Fde0f/Oxide.js

```diff
@@ -0,0 +1,331 @@
+/* eslint-disable no-console */
+
+import React, { useState } from 'react'
+import { Grid, TextField, Typography, Button } from '@material-ui/core'
+import Chip from '@material-ui/core/Chip'
+import SvgIcon from '@material-ui/core/SvgIcon'
+import Avatar from '@material-ui/core/Avatar'
+import OxideIcon from '@material-ui/icons/InvertColors';
+import TimerIcon from '@material-ui/icons/HourglassEmpty';
+import UsersIcon from '@material-ui/icons/People';
+import WagerIcon from '@material-ui/icons/LocalDrink';
+import DiceIcon from '@material-ui/icons/Casino';
+import WinnerIcon from '@material-ui/icons/Whatshot';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+
+import { createMuiTheme } from '@material-ui/core/styles';
+
+import FingerprintIcon from '@material-ui/icons/Fingerprint';
+import { withStyles } from '@material-ui/core/styles';
+import StarIcon from '@material-ui/icons/StarBorder';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+
+import { useAccountEffect, useWeb3Context } from 'web3-react/hooks'
+import { toDecimal, fromDecimal } from 'web3-react/utilities'
+
+import { useSnowflakeBalance, useGenericContract } from '../../../../common/hooks'
+import TransactionButton from '../../../common/TransactionButton'
+
+import { ABI } from './index'
+
+import './Oxide.css'
+
+const CustomTableCell = withStyles(theme => ({
+  head: {
+    backgroundColor: '#0971f5',
+    color: theme.palette.common.white,
+    fontSize: 20,
+
+  },
+  body: {
+    fontSize: 14,
+  },
+}))(TableCell);
+
+
+export default function Oxide ({ ein }) {
+  const context = useWeb3Context()
+  const [activePot, setPot]  = useState(0)
+  const [activeRound, setRound]  = useState(0)
+  const [activeWinner, setWinner]  = useState(0)
+  const [oxideBalance, setOxide]  = useState(0)
+  const [committedWager, setWager]  = useState(0)
+  const [activePunters, setPunters]  = useState(0)
+  const [open, setOpen] = useState(false);
+  const [leaderboardData, setLeaderboard]  = useState([])
+  const oxideContract = useGenericContract('0x2930Cf9EE8E03C3E06Fa1828cCD8E371323Fde0f', ABI)
+  const snowflakeBalance = useSnowflakeBalance(ein)
+
+  function refreshLeaderboard(_round)  {
+    oxideContract.getPastEvents("scoreLog", { fromBlock: 0, toBlock: 'latest' })
```

Comment:
```suggestion
    oxideContract.getPastEvents("scoreLog", { fromBlock: 3795013, toBlock: 'latest' })
```

Reply:
This is the block height that snowflake was deployed at, and will speed up the event fetching stage.